kolibri/core/auth/management/commands/bulkimportusers.py

```diff
@@ -656,9 +656,7 @@ def handle_async(self, *args, **options):
                     Classroom.objects.create(
                         name=classroom.name, parent=classroom.parent
                     )
-                # clear users from classes to be updated:
-                update_classes = [c.id for c in db_update_classes]
-                Membership.objects.filter(collection__in=update_classes).delete()
+
```

Comment:
two questions, just to double check:

1. will this still handle the scenario where:
    * a learner is enrolled
    * export a CSV
    * edit the learner in the CSV to have a blank enrollment column
    * import CSV
    * the learner should be removed from the class
2. does this behavior affect both learner enrollment and coach assignment?

Reply:
1. No, the learner won't be removed, that's adding another level of complexity. If needed we can handle it, but I'd do it in a different PR, as it's a total different history.
2. Yes, both assignment and enrollment are managed the same way